umap/static/umap/js/modules/data/layer.js

```diff
@@ -802,13 +802,11 @@ export class DataLayer {
       this
     )
     if (this.umap_id) {
-      const download = DomUtil.createLink(
-        'button umap-download',
-        advancedButtons,
-        translate('Download'),
-        this._dataUrl(),
-        '_blank'
-      )
+      const download = Utils.loadTemplate(`
+        <a class="button" href="${this._dataUrl()}" target="_blank">
```

Comment:
Instead of the `target` attribute, I think a `download` one with an explicit name would be more interesting for the user.